Core/src/org/sleuthkit/autopsy/coreutils/Logger.java

```diff
@@ -90,17 +90,21 @@ private static FileHandler createFileHandlerWithTraces(String logDirectory) {
             fileHandler.setFormatter(new Formatter() {
                 @Override
                 public String format(LogRecord record) {
-                    if (record.getThrown() != null) {
+                    Throwable thrown = record.getThrown();
+                    if (thrown != null) {
```

Comment:
Doing so creates extra whitespace (because there is a \n after stackTrace), which is undesirable.